lightning: refine description about checksum-via-sql (#14241)
pingcap/docs: tidb-lightning/tidb-lightning-configuration.md

# This setting controls the log level of the TiDB library.
log-level = "error"

> # Sets the TiDB session variable to speed up the Checksum and Analyze operations. Note that if checksum-via-sql is set to "true", you need to execute the ADMIN CHECKSUM TABLE <table> SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency = 15` and `checksum-table-concurrency = 2` will not take effect.

Reviewer: ```suggestion
# Sets the TiDB session variable to speed up the Checksum and Analyze operations. Note that if checksum-via-sql is set to "true", Lightning will execute the ADMIN CHECKSUM TABLE <table> SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency = 15` and `checksum-table-concurrency = 2` will not take effect.
```

> # Sets the TiDB session variable to speed up the Checksum and Analyze operations. Note that if checksum-via-sql is set to "true", TiDB Lightning will execute the ADMIN CHECKSUM TABLE <table> SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency = 15` and `checksum-table-concurrency = 2` will not take effect.

Reviewer: ```suggestion
# Sets the TiDB session variable to speed up the Checksum and Analyze operations. Note that if checksum-via-sql is set to "true", TiDB Lightning will execute the ADMIN CHECKSUM TABLE <table> SQL statement to perform the Checksum operation on TiDB. In this case, the following parameters `distsql-scan-concurrency ` and `checksum-table-concurrency ` will not take effect.
```

# For backward compatibility, bool values "true" and "false" are also allowed for this field.
# "true" is equivalent to "required" and "false" is equivalent to "off".
checksum = "required"
# Specifies whether the ADMIN CHECKSUM TABLE <table> operation is executed via TiDB.
# The default value is "false", which means that the ADMIN CHECKSUM TABLE <table> command is sent to TiKV for execution via TiDB Lightning.
# It is recommended that you set this value to "true" to make it easier to locate the problem if checksum fails.

> # Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` parameter in TiDB.

Reviewer: add the  link for `tidb_checksum_table_concurrency` https://docs.pingcap.com/zh/tidb/stable/system-variables#tidb_checksum_table_concurrency

> # Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` parameter in TiDB.

Author: ```suggestion
# Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` (https://docs.pingcap.com/zh/tidb/stable/system-variables#tidb_checksum_table_concurrency) parameter in TiDB.
```

> # checksum-via-sql = false

Author: ```suggestion
checksum-via-sql = "false"
```

> # Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` (https://docs.pingcap.com/tidb/stable/system-variables#tidb_checksum_table_concurrency) parameter in TiDB.

Reviewer 2: ```suggestion
# Meanwhile, if you want to adjust concurrency when this value is "true", you need to set the `tidb_checksum_table_concurrency` variable in TiDB (https://docs.pingcap.com/tidb/stable/system-variables#tidb_checksum_table_concurrency).
```
